actors/common/src/main/java/org/sunbird/learner/util/EkStepRequestUtil.java

```diff
@@ -38,9 +38,9 @@ private EkStepRequestUtil() {}
     String response = "";
     JSONObject jObject;
     try {
-      String baseSearchUrl = System.getenv(JsonKey.EKSTEP_BASE_URL);
+      String baseSearchUrl = System.getenv(JsonKey.SEARCH_SERVICE_API_BASE_URL);
```

Comment:
ProjectUtil has method to get config value directly from either env or properties file.